eap-xp4/runtime-image/rel-jdk17-overrides.yaml

```diff
@@ -0,0 +1,42 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp4-openjdk17-runtime-openshift-rhel8"
+
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-xp4-openjdk17-runtime-openshift-rhel8-container"
+
+envs:
+     - name: DISABLE_LEGACY_SECURITY
+       value: "true"
+     - name: AB_JOLOKIA_OFF
+       value: "true"
+     - name: CONFIGURE_ELYTRON_SSL
+       value: "true"
+     - name: SSO_FORCE_LEGACY_SECURITY
+       value: "false"
+
+modules:
+      repositories:
+          - name: jdk-modules
+            path: ../../modules                  
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "17"
```

Comment:
What is the relation of these overrides to eap-xp4-jdk17/runtime-image/rel-jdk17-overrides.yaml?

Reply:
@xstefank I forgot to remove the file, this file was for JDK17 with SSO